### PR TITLE
feat(business): toast notification

### PR DESCRIPTION
**Pull request by vlanz**
Implement simple toast notification and add examples and documentation

### CODE REVIEW

File: src/angular-business/notification-toast/notification-toast.ts [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+  /** The component that should be rendered as the snack bar's simple component. */
+  protected _simpleSnackBarComponent: Type<TextOnlyNotificationToast> = SimpleNotificationComponent;
+
+  /** The container component that attaches the provided template or component. */
+  protected _snackBarContainerComponent: Type<
+    NotificationToastContainerComponent
+  > = NotificationToastContainerComponent;
+
+  /** The CSS class to applie for handset mode. */
+  protected _handsetCssClass = 'mat-snack-bar-handset';
```
Comment (jeripeierSBB) [inline src/angular-business/notification-toast/notification-toast.ts:66]: mat- ? :)

File: src/showcase/app/business/business-examples/notification-toast-examples/notification-toast-component-example/notification-toast-component-example.component.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  }
+}
+
+@Component({
+  selector: 'sbb-example-toast',
+  template: `<span class="example-style">SBB CFF FFS</span>`,
+  styles: [
+    `
+      .example-style {
+        color: darkred;
```
Comment (jeripeierSBB) [inline src/showcase/app/business/business-examples/notification-toast-examples/notification-toast-component-example/notification-toast-component-example.component.ts:22]: for me, this color looks weird, maybe take an sbb color? oder just style another thing than color :)

File: src/angular-business/notification-toast/notification-toast-container.component.scss [hunk 3]
```diff
@@ -0,0 +1,77 @@
+$sbbBusiness: true;
```
Comment (jeripeierSBB) [inline src/angular-business/notification-toast/notification-toast-container.component.scss:1]: I see a mix of px and rem. As intended?

File: src/angular-business/notification-toast/notification-toast.spec.ts [hunk 4]
```diff
@@ -0,0 +1,947 @@
...
+    );
+
+    const messageElement = overlayContainerElement.querySelector(
+      'sbb-notification-toast-container'
+    )!;
+    expect(messageElement.textContent).toContain(
+      simpleMessage,
+      `Expected the notification toast message to be '${simpleMessage}'`
+    );
+    expect(overlayContainerElement.querySelector('button.mat-button')).toBeNull(
```
Comment (jeripeierSBB) [inline src/angular-business/notification-toast/notification-toast.spec.ts:277]: mat-button

File: src/angular-business/notification-toast/notification-toast.ts [hunk 5]
```diff
@@ -0,0 +1,319 @@
...
+
+      // We can't pass this via the injector, because the injector is created earlier.
+      notificationRef.instance = contentRef.instance;
+    }
+
+    // Subscribe to the breakpoint observer and attach the notification-handset class as
+    // appropriate. This class is applied to the overlay element because the overlay must expand to
+    // fill the width of the screen for full width notifications.
+    this._breakpointObserver
+      .observe(Breakpoints.HandsetPortrait)
```
Comment (jeripeierSBB) [inline src/angular-business/notification-toast/notification-toast.ts:221]: breakpoint from material, shouldn't we use our own?

**Review by jeripeierSBB — CHANGES_REQUESTED** [review]
I think it should be rebased onto master because of the bazel file changes

File: src/showcase/app/business/business-examples/notification-toast-examples/notification-toast-component-example/notification-toast-component-example.component.ts (hunk 2 above)
Comment (kyubisation) [reply src/showcase/app/business/business-examples/notification-toast-examples/notification-toast-component-example/notification-toast-component-example.component.ts:22]: Changed to font style italic

File: src/angular-business/notification-toast/notification-toast-container.component.scss (hunk 3 above)
Comment (kyubisation) [reply src/angular-business/notification-toast/notification-toast-container.component.scss:1]: Changing to rem

File: src/angular-business/notification-toast/notification-toast.ts (hunk 5 above)
Comment (kyubisation) [reply src/angular-business/notification-toast/notification-toast.ts:221]: Absolutely. Missed this.

File: src/angular-business/notification-toast/notification-toast.ts [hunk 6]
```diff
@@ -0,0 +1,320 @@
...
+  /** The component that should be rendered as the snack bar's simple component. */
+  protected _simpleSnackBarComponent: Type<TextOnlyNotificationToast> = SimpleNotificationComponent;
+
+  /** The container component that attaches the provided template or component. */
+  protected _snackBarContainerComponent: Type<
+    NotificationToastContainerComponent
+  > = NotificationToastContainerComponent;
+
+  /** The CSS class to applie for handset mode. */
+  protected _handsetCssClass = 'sbb-notification-toast-handset';
```
Comment (jeripeierSBB) [inline src/angular-business/notification-toast/notification-toast.ts:67]: wording: handset to mobile?

File: src/angular-business/notification-toast/notification-toast.ts [hunk 7]
```diff
@@ -0,0 +1,320 @@
...
+   */
+  openFromTemplate(
+    template: TemplateRef<any>,
+    config?: SbbNotificationToastConfig
+  ): SbbNotificationToastRef<EmbeddedViewRef<any>> {
+    return this._attach(template, config);
+  }
+
+  /**
+   * Opens a snackbar with a message and an optional action.
```
Comment (jeripeierSBB) [inline src/angular-business/notification-toast/notification-toast.ts:122]: generally rename snackbar to toast? (there also other occurences)

File: src/angular-business/notification-toast/notification-toast.ts [hunk 8]
```diff
@@ -0,0 +1,320 @@
...
+    }
+
+    if (config.announcementMessage) {
+      this._live.announce(config.announcementMessage, config.politeness);
+    }
+  }
+
+  /**
+   * Creates a new overlay and places it in the correct location.
+   * @param config The user-specified snack bar config.
```
Comment (jeripeierSBB) [inline src/angular-business/notification-toast/notification-toast.ts:279]: another snackbar occurrence, here in two words (snack bar)

File: src/angular-business/notification-toast/notification-toast.ts [hunk 9]
```diff
@@ -0,0 +1,320 @@
...
+   */
+  private _createOverlay(config: SbbNotificationToastConfig): OverlayRef {
+    const overlayConfig = new OverlayConfig();
+    overlayConfig.direction = 'ltr';
```
Comment (jeripeierSBB) [inline src/angular-business/notification-toast/notification-toast.ts:283]: isn't 'ltr' default?

**Review by jeripeierSBB — COMMENTED** [review]
Generally ok for me, a few questions left

**Review by jeripeierSBB — APPROVED** [review]
LGTM :)

**Pull request merged**